SystemExecutionerTest can fail intermittently

The following test failure occurred when running checks for #182 :

```
Error:  Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.317 s <<< FAILURE! - in org.kiwiproject.dropwizard.util.startup.SystemExecutionerTest
Error:  org.kiwiproject.dropwizard.util.startup.SystemExecutionerTest.shouldExitBeforeGivenWaitTime_WhenWaitingThreadInterrupted  Time elapsed: 0.108 s  <<< FAILURE!
java.lang.AssertionError: 
[Elapsed millis must be greater than 100] 
Expecting actual:
  100L
to be greater than:
  100L

	at org.kiwiproject.dropwizard.util.startup.SystemExecutionerTest.shouldExitBeforeGivenWaitTime_WhenWaitingThreadInterrupted(SystemExecutionerTest.java:105)
```

I don't remember ever seeing this happen before but since it did, it is possible that the action is happening so fast that the elapsed time _equals_ the `killerSleepTimeMillis` of 100 milliseconds. The test assertion currently uses `isGreaterThan(killerSleepTimeMillis)`. Because we know this _can_ happen (again, since it actually _did_ happen), we should change the assertion to greater than _or equal to_.

--- a/src/test/java/org/kiwiproject/dropwizard/util/startup/SystemExecutionerTest.java
+++ b/src/test/java/org/kiwiproject/dropwizard/util/startup/SystemExecutionerTest.java
@@ -100,9 +100,11 @@ class SystemExecutionerTest {
                 .describedAs("Execution strategy exit() should have been called")
                 .isTrue();
 
-        assertThat(TimeUnit.NANOSECONDS.toMillis(elapsedNanos))
-                .describedAs("Elapsed millis must be greater than %d", killerSleepTimeMillis)
-                .isGreaterThan(killerSleepTimeMillis);
+        var elapsedMillis = TimeUnit.NANOSECONDS.toMillis(elapsedNanos);
+        LOG.info("Actual elapsed time: {} nanoseconds ; {} milliseconds", elapsedNanos, elapsedMillis);
+        assertThat(elapsedMillis)
+                .describedAs("Elapsed millis must be at least %d", killerSleepTimeMillis)
+                .isGreaterThanOrEqualTo(killerSleepTimeMillis);
 
         executorService.shutdown();
         await().atMost(ONE_SECOND).until(executorService::isShutdown);

